src/bin/find.js

```diff
@@ -8,6 +8,7 @@ var options = {
   getAllAvailableRules: ['all-available', 'a'],
   getUnusedRules: ['unused', 'u'],
   n: ['no-error'],
+  noCore: ['nocore', 'C'],
```

Comment:
Just saw your command to remove `-C`, will do. But the previous "investigation" still holds true for `--no-error`, which is currently bugged.